libraries/AP_Baro/AP_Baro.h

```diff
@@ -177,6 +180,8 @@ class AP_Baro
         float altitude;                 // calculated altitude
         AP_Float ground_pressure;
         float p_correction;
+        uint8_t can_manager;
+        uint8_t uavcan_node;
```

Comment:
AP_UAVCAN is actually tracking all baros (on it's network [manager]). When new baro message arrives, it calls register_uavcan_baro in AP_Baro to check if there was such baro before or it has to be registered.
As long as there is only one AP_UAVCAN instance, it can be made to keep all registered baros inside UAVCAN, but when AP_Baro can receive the call from many instances of UAVCAN, only AP_Baro can know from which one it is coming and so take actions on registering.